index.md

```diff
@@ -10,18 +10,45 @@ layout: standard-page
 Good humored, cooperative, competitive, consensual torture, for the purposes of
 exploring strategies, design, progress and history of gaming
 
-### Members
-* There is no single leader of the Zargon community. Zargon is a community run
-  effort. All members are encouraged to participate in this effort.
-* You may invite new members to join in gaming, but they are your
-  responsibility.
-* New invitees should come to provisional games (e.g., Friday games, or similar
-  virtual games) for the sake of the invitee and the community.
-* If community members at large do not like a guest then that guest may not be
-  invited to future events.
+## Membership
+There is no single leader of the Zargon community. Zargon is a community run effort. All members are encouraged to participate in this effort.
+
+The following guidelines are intended to allow us to maintain an inclusive environment where all members feel comfortable gaming with any other member of the group rather than fracturing in cliques. They are enumerated in detail so as to mitigate subjectivity in role assignment.
+
+### Obligations
 * Members are expected to maintain basic personal hygiene.
 * Members are expected to be gracious and respect additional restrictions of
-  venue (e.g. no smoking inside; no young children; allergy considerations).
+  venue
+    - e.g. no smoking inside; no young children; allergy considerations.
+
+### Active Members
+* An active member is one who regularly does one or more of the following
+  - Attends scheduled gaming or social events
+  - Paricipates in virtual conversations
+* Active members are noted by the **Zargonaut** role in the group discord.
+
+### Contributing Members
+* A contributing member is one who regularly does one or more of the following
+  - Schedules game sessions
+  - Schedules social events
+  - Brings new games for the group to try
+  - Teaches games at gaming evens
+  - Creates content to aid the group (e.g. rules handouts, character illustions, games storage)
+* Contributing members are noted by the **Adept** role in the group discord.
+
+### New Members
+* Any active member in good standing may invite non-member to join social
+  or gaming events, but take responsibility for their behavior.
+  - In the case of a virtual event, they will be assigned the **Attendee** role in the group discord.
```

Comment:
- **Attendee** roles are temporary and should be removed at the end of the specific event or on a regular basis

^ otherwise attendees basically have normal access

Reply:
I'd think they would only get access to the voice channels and a single text channel. But I'll add that